pyi.py

```diff
@@ -125,26 +125,39 @@ def all_equal(iterable: Iterable[object]) -> bool:
 }
 
 # Y023: Import things from typing instead of typing_extensions
-# if they're available from the typing module on 3.7+
+# if they're available from the typing module on 3.8+
 _BAD_TYPINGEXTENSIONS_Y023_IMPORTS = frozenset(
     {
+        "Any",
         "AnyStr",
         "BinaryIO",
+        "Final",
         "ForwardRef",
         "Generic",
         "IO",
+        "Literal",
         "Protocol",
         "TextIO",
         "runtime_checkable",
         "NewType",
+        "SupportsAbs",
+        "SupportsComplex",
+        "SupportsFloat",
+        "SupportsIndex",
+        "SupportsInt",
+        "SupportsRound",
+        "TypedDict",
+        "TypeVar",
+        "final",
         "overload",
         "NoReturn",
         # ClassVar deliberately omitted,
```

Comment:
Is this comment still accurate? There are now several others in the list that can be parameterized (Generic, Literal, Protocol).

Reply:
I think `ClassVar` is _probably_ the only one worth special-casing in error messages. But if you'd like to PR something up special-casing some of the others in error messages as well, feel free :)